lib/project_copier.rb

```diff
@@ -0,0 +1,30 @@
+class ProjectCopier
+  EXCLUDE_ATTRIBUTES = [ :classifications_count, :launched_row_order, :beta_row_order ].freeze
+  INCLUDE_ATTRIBUTES = [  :project_contents,
```

Comment:
These aren't really attributes but associations. Should the variable name reflect that?

Reply:
✔️